src/applications/static-pages/simple-forms/40-0247/App.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+const App = ({ formEnabled }) => {
+  if (formEnabled === undefined) {
+    return <va-loading-indicator message="Loading..." />;
+  }
+
+  if (formEnabled) {
+    return (
+      <>
+        <p>You can apply for a PMC in any of these 5 ways.</p>
+        <h3>Option 1: Apply online</h3>
+        <a
+          className="vads-c-action-link--blue"
+          href="/burials-memorials/memorial-items/presidential-memorial-certificates/request-certificate-form-40-0247"
+        >
+          Apply for a PMC online
+        </a>
+        <h3>Option 2: Mail your application</h3>
+        <p>
+          First, complete the Presidential Memorial Certificate Request Form (VA
+          Form 40-0247).
+        </p>
+        <p>
+          <va-link
+            href="/find-forms/about-form-40-0247"
+            text="Get VA Form 40-0247 to download"
+          />
+        </p>
+        <p>
+          Mail your application, along with copies of the Veteran’s death
+          certificate and DD214 or other discharge documents, to this address:
+        </p>
+        <p>
+          NCA FP Evidence Intake Center
+          <br />
+          PO Box 5237
+          <br />
+          Janesville, WI 53547
+        </p>
+        <p>
+          <strong>Note</strong>: Please don’t send original documents since we
+          can’t return them to you.
+        </p>
+        <p>
+          <va-link
+            href="https://www.cem.va.gov/CEM/hmm/discharge_documents.asp"
+            text="Review our list of military discharge documents"
+          />
+        </p>
+        <h3>Option 3: Upload your application</h3>
+        <p>
+          First, complete the Presidential Memorial Certificate Request Form (VA
+          Form 40-0247).
+        </p>
+        <p>
+          <va-link
+            href="/find-forms/about-form-40-0247"
+            text="Get VA Form 40-0247 to download"
+          />
+        </p>
+        <p>
+          Then, upload your completed application and supporting documents on
+          the AccessVA website using the QuickSubmit tool.
+        </p>
+        <p>
+          If it’s your first time signing into this tool, you’ll need to
```

Comment:
Could we add a space between in and to? It should be "...signing in to this..." @tlei123

Reply:
I sent it the wrong way in the initial copy. Sorry about that!

Reply:
@megzehn @tbaker1026 , yes, and done.